kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/RecordSerDe.java

```diff
@@ -10,9 +12,17 @@
 public interface RecordSerDe {
 
   @Value
+  @Builder
   class DeserializedKeyValue {
     @Nullable String key;
     @Nullable String value;
+    @Nullable MessageFormat keyFormat;
+    @Nullable MessageFormat valueFormat;
+    @Nullable Integer keySize;
+    @Nullable Integer valueSize;
+    @Nullable String keySchemaId;
```

Comment:
is it intended to make keySchemaId & valueSchemaId strings? Was it done for future extension?

Reply:
Yes, I made them strings for the future, e.g. we will support something different than confluent schema registry.

Reply:
Fixed, please review